examples/multi-bot/bot-facebook.js

```diff
@@ -23,7 +23,15 @@ var controller = Botkit.facebookbot({
 
 var bot = controller.spawn();
 controller.hears('(.*)', 'message_received', function(bot, message) {
-  bot.reply(message, message.watsonData.output.text.join('\n'));
+  if (message.watsonError) {
+    console.log(message.watsonError);
+    bot.reply(message, "I'm sorry, but for technical reasons I can't respond to your message");
```

Comment:
We should just send the error message back to the user instead of writing a custom error message.

```bot.reply(message, message.watsonData.output.text.join('\n'));

Reply:
Changes committed, assuming watsonError objects have description and/or error fields. I'm a bit confused with https://www.ibm.com/watson/developercloud/conversation/api/v1/#error-handling - there "code" and "message" are mentioned as elements of the error object, or is that a different part of the system?